frappe/custom/doctype/customize_form/customize_form.js

```diff
@@ -49,6 +49,10 @@ frappe.ui.form.on("Customize Form", {
 			if (grid_row.doc && grid_row.doc.fieldtype == "Section Break") {
 				$(grid_row.row).css({ "font-weight": "bold" });
 			}
+
+			if (grid_row.doc.is_custom_field && !grid_row.row.hasClass('highlight')) {
```

Comment:
```suggestion
			if (grid_row.doc.is_custom_field && !grid_row.row.hasClass('highlight') && !grid_row.doc.is_system_generated) {
```

Since [this](https://github.com/frappe/frappe/pull/16278) got merged, let's highlight only user-generated custom fields?

Might need other changes as well.

Reply:
or should we highlight the user-generated custom fields with a different color?
(In this [PR](https://github.com/frappe/frappe/pull/16278) The patch has updated the custom fields but the cache is not cleared)

Reply:
TBH, not sure if that patch should be done at all. Many users create Custom Field and Property Setter as Administrator. Color: I think same is fine. Because it's behavior should also be the same. Just like you cannot change specific properties in the standard fields, you should also not be able to change these in system generated fields.